miso-dto/src/main/java/uk/ac/bbsrc/tgac/miso/dto/Dtos.java

```diff
@@ -702,6 +712,60 @@ private static SampleTissue toTissueSample(SampleTissueDto from) {
     return to;
   }
 
+  private static SampleTissueProcessingDto asTissueProcessingSampleDto(SampleTissueProcessing from) {
+    SampleTissueProcessingDto dto = null;
+    if (from.getClass() == SampleCVSlideImpl.class) {
```

Comment:
getClass comparison is required for Dtos because of the way their inheritance works (analyte is also a tissue and an identity for the purpose of including parent data), but instanceof should be used here (and typically for the non-dto models), and check vs. interfaces for flexibility. e.g. we could add another implementation of SampleCVSlide that this method should still work for, or we could subclass SampleCVSlideImpl (however unlikely)